apps/condo/domains/organization/integrations/sbbol/SbbolFintechApi.js

```diff
@@ -173,11 +296,17 @@ class SbbolFintechApi extends SbbolRequestApi {
                 'Content-Type': 'application/json',
             },
         })
-        const jsonData = JSON.parse(data)
-        if (statusCode === 201) {
-            return { data: jsonData }
-        } else {
-            return { error: jsonData }
+
+        try {
+            const jsonData = JSON.parse(data)
+
+            if (statusCode === 201) {
+                return { data: jsonData }
+            } else {
+                return { error: jsonData }
```

Comment:
statusCode ?

Reply:
it seems that the statusCode is not needed for the logic further